### DISCUSSION TITLE
Is it possible to calculate the time varying deformation in the satellite orbit

### DISCUSSION THREAD

**Discussion by donnie-jyt** [groops-devs/groops#222] [General]
I know GROOPS is a very powerful tool to calculate kinds of variables related to the Earth. However, I'd like to ask if I wanna calculate the tide (Ocean tide and Solid tide) or other time-varying deformation related to the Earth and convert these values to the satellite orbit, how should I use the GROOPS?
For example, I wanna calculate the ocean tide deformation in the GOCE satellite orbit, how should I use it?

**Comment by tmayerguerr** [comment]
Hi,
You can model all effects within [SimulateOrbit](https://groops-devs.github.io/groops/html/SimulateOrbit.html). If you are interested in the accelerations in the satellite reference frame you can use [SimulateAccelerometer](https://groops-devs.github.io/groops/html/SimulateAccelerometer.html). All time varying forces can be set up in [forces](https://groops-devs.github.io/groops/html/forcesType.html). Hope this answers your question.
regards
Torsten
